### ISSUE TITLE
Call to undefined method Yajra\Oci8\Schema\Grammars\OracleGrammar::typeJson()

### ISSUE THREAD

**Issue by adrianharabula** [yajra/laravel-oci8#283]
### Summary of problem or feature request

Cannot create a migration with a json type field using `$table->json('column_name');`

### Code snippet of problem

```php
        Schema::create('characterizable', function (Blueprint $table) {
            $table->integer('characteristic_id')->unsigned();
            $table->foreign('characteristic_id')->references('id')->on('characteristics');
            $table->json('characteristic_values');
            $table->integer('characterizable_id')->unsigned();
            $table->string('characterizable_type');
            $table->timestamps();

            // TODO: maybe add indexes for this table
        });
```
### System details

- PHP Version 7.1.4
- Laravel Version 5.4.18
- Laravel-OCI8 Version 5.4.4

**Comment by yajra** [comment]
I think JSON is not supported by the package and Oracle 11g and below?

**Comment by yajra** [comment]
Fixed via #334
